cmd/ctl/pkg/create/certificaterequest/certificaterequest.go

```diff
@@ -319,10 +319,13 @@ func buildCertificateRequest(crt *cmapi.Certificate, pk []byte, crName string) (
 			Duration:  crt.Spec.Duration,
 			IssuerRef: crt.Spec.IssuerRef,
 			IsCA:      crt.Spec.IsCA,
-			Usages:    crt.Spec.Usages,
 		},
 	}
 
+	if crt.Spec.EncodeUsagesInRequest {
```

Comment:
this raises the question here of what to default to. Given we added this in v1.0 we should always add usages into the CSR unless disabled. Doing this with bools means we have to revert the working `doNotIncluseUsagesInCSR` or something similar... (i hate this proposed name, don't use it)

Reply:
Right, so I should invert the logic here since the new field will default to `false`.

Reply:
maybe `excludeUsagesFromRequest` or `ignoreUsagesInRequest`?

Reply:
Really hate double negatives on bool values :sweat_smile: 
I think "EncodeUsagesInRequest" reads a lot better.

Maybe change this to a *bool? What do you think @meyskens?

We default to true if crt.Spec.EncodeUsagesInRequest == nil

Reply:
Totally agree on double negatives, it's very confusing to use.

The `*bool` makes the interface easier to use, but the code harder to read :wink: